### ISSUE TITLE
color scheme in 3.0 needs to be changed to allow for better customizing

### ISSUE THREAD

**Issue by garyruskin** [belaviyo/dark-theme#19]
The 'black' search box, footer and header, in 3.0,  were visually better in 2.8, when they matched the google background color. It would be better to have the option to select color of search box and other areas that are now limited to dark black only.

**Comment by belaviyo** [comment]
Try 0.3.0.1 release.

**Comment by garyruskin** [comment]
Just tried v3.01. Same issue as in v3.0, with black background in the
search bar and bottom of page. Reverting to v2.8

Using Firefox Mozilla, Windows 10 platform.

Does 3.01 have any options that 3.0 didn't have? Doesn't appear to make any
difference - at least on my PCs.

--
t37anna@gmail.com

**Comment by belaviyo** [comment]
in v0.3.0.1, the header and footer respects the light dark color of the options page.

**Comment by garyruskin** [comment]
yes, the light dark color does respect the color of the options page, but
only on the search results page. It does not respect the color of the
search bar itself on the google home page (which defaults to a background
color of black, as does the footer) of the main google search page.
Changing the light dark color does not seem to affect the main page in any
way.

--
t37anna@gmail.com
